ConsoleGUI/Controls/ListView.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using ConsoleGUI.Data;
+using ConsoleGUI.Input;
+using ConsoleGUI.UserDefined;
+
+namespace ConsoleGUI.Controls
+{
+    public class ListView : SimpleControl, IInputListener
+    {
+        public Color TextColor { get; set; } = Color.White;
+        public Color SelectedTextColor { get; set; } = Color.Black;
```

Comment:
I wonder if controling the `TextColor` and the `SelectedTextColor` is enough. Maybe it would be a good idea to also allow for setting the background color of the selected text and `VerticalScrollPanel` properties?

Reply:
I was more thinking about a property that would control the background color of a selected row. I fully agree that the background color of regular rows should be controlled by wrapping this control in a border.